### DISCUSSION TITLE
Does develop branch contain version 1.11 of human1 or is still at the 1.10?

### DISCUSSION THREAD

**Discussion by PkiwiBird** [SysBioChalmers/Human-GEM#377] [Q&A] [answered]
Does develop branch contain version 1.11 of human1 or is it still at 1.10? (23/3/2022)

**Comment by JonathanRob** [comment, marked as answer]
The `develop` branch is always either _ahead of_ or _equivalent to_ the `main` branch, so at the moment the `development` branch exists somewhere _after_ version 1.11. Since official releases are made only from the `main` branch, the model on the `development` branch never actually has a version number associated with it.
